src/components/Image/src/Image.vue

```diff
@@ -141,6 +146,7 @@ export default {
 		style() {
 			return {
 				'--image-height': `${this.height}px`,
+				'object-fit': this.objectFit,
```

Comment:
using the `.Image &.shape_arch` style as reference, should this be plugged in the same way as `--image-height` so that you have
```css
.Image {
   object-fit: var(--object-fit);
```

Reply:
Yeah that makes sense, just applied that change